README.md

```diff
@@ -42,6 +43,30 @@ Prequisites:
   redis-cli ping  # Should respond with PONG
   rake db:setup
   ```
+Ubuntu:
+- Update system
+  ```shell
+  sudo apt update
+  ```
+- Install redis server
+  ```shell
+  sudo apt install redis-server
+  ```
+- Update configuration
+  ```shell
+  sudo sh -c 'echo "supervised systemd" >> /etc/redis/redis.config'
```

Comment:
What is this line `sudo sh -c 'echo "supervised systemd" >> /etc/redis/redis.config'` doing/why would someone need to run this?

Reply:
This command appends the line `supervised systemd` to the redis configuration file (`/etc/redis/redis.config`), which creates a [systemd](https://en.wikipedia.org/wiki/Systemd) service that runs the redis server instance. It allows you to start, stop, check the status of, restart the redis-server using `systemctl` commands like `systemctl start redis-server`.